lib/_http_server.js

```diff
@@ -762,7 +762,10 @@ const requestHeaderFieldsTooLargeResponse = Buffer.from(
 function socketOnError(e) {
   // Ignore further errors
   this.removeListener('error', socketOnError);
-  this.on('error', noop);
+
+  if (!this.listenerCount('error') === 0) {
```

Comment:
```suggestion
  if (this.listenerCount('error') === 0) {
```
@mcollina's suggestion was half applied :eyes: